source/lambda/main.py

```diff
@@ -30,21 +30,6 @@
 LOG_STREAM = "InstanceScheduler-{:0>4d}{:0>2d}{:0>2d}"
 
 
-# load models for services that have not have their latest models deployed to Lambda
-def load_models():
-    cdw = os.getcwd()
-    models = os.path.join(cdw, "models")
-    aws_data_path = os.getenv("AWS_DATA_PATH", None)
-    if aws_data_path is not None:
-        aws_data_path = ":".join([aws_data_path, models])
-    else:
-        aws_data_path = models
-    os.environ["AWS_DATA_PATH"] = aws_data_path
-
-
-load_models()
-
-
```

Comment:
I presume this has been confirmed as an unused method? It is not clear to me how it pertains to the old retry logic